Cargo.toml

```diff
@@ -31,8 +31,9 @@ version = "0.2"
 path = "./dotrix_ui"
 optional = true
 
-[dev-dependencies.cgmath]
-version = "0.17"
+[dependencies.dotrix_math]
+version = "0.2"
+path = "./dotrix_math"
```

Comment:
You should also re-export it in lib.rs, just as it is for ui, so one can use the crate as
`use dotrix::math;`